syz-manager/manager.go

```diff
@@ -57,7 +57,8 @@ var (
 		"	the process exits and the exit status indicates success/failure.\n"+
 		"	If the kernel oopses during testing, the report is saved to workdir/report.json.\n"+
 		" - corpus-triage: triage corpus and exit\n"+
-		"	This is useful mostly for benchmarking with testbed.\n")
+		"	This is useful mostly for benchmarking with testbed.\n"+
+		" - corpus-run: continuously run the corpus programs.\n")
```

Comment:
This effectively does what tools/syz-crush does. Maybe let's remove syz-crush now?

Reply:
Strictly speaking, `tools/syz-crush` does a bit different thing -- it executes the same set of programs on every VM. If we want to run it as quickly as we can and collect all possible crashes, we want to evenly spread those programs over many VMs.